<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="accessibleName">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;DATA&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1171</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Data</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="datafilepath_lineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>1031</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="select_data_file_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose file</string>
     </property>
    </widget>
    <widget class="QFrame" name="horizontalFrame_2">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>180</y>
       <width>471</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="set_saved_data_config_btn">
        <property name="text">
         <string>Save data configuration to:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dataconfig_lineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="gridGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>1151</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Components:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QCheckBox" name="checkBox_Tyz">
        <property name="text">
         <string>Tyz</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QLineEdit" name="line_abs_err_Tzz">
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="line_rel_err_V"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="line_abs_err_gz"/>
      </item>
      <item row="2" column="7">
       <widget class="QLineEdit" name="line_rel_err_Txz"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Error (relative):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="line_abs_err_gx"/>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="line_abs_err_gy"/>
      </item>
      <item row="2" column="8">
       <widget class="QLineEdit" name="line_rel_err_Tyz"/>
      </item>
      <item row="0" column="10">
       <widget class="QCheckBox" name="checkBox_Txy">
        <property name="text">
         <string>Txy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBox_gx">
        <property name="text">
         <string>gx</string>
        </property>
       </widget>
      </item>
      <item row="3" column="8">
       <widget class="QLineEdit" name="line_abs_err_Tyz"/>
      </item>
      <item row="3" column="11">
       <widget class="QLineEdit" name="line_abs_err_Tyy"/>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="line_rel_err_gy"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="line_abs_err_V">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QCheckBox" name="checkBox_Tzz">
        <property name="text">
         <string>Tzz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="line_rel_err_gx"/>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_gz">
        <property name="text">
         <string>gz</string>
        </property>
       </widget>
      </item>
      <item row="3" column="9">
       <widget class="QLineEdit" name="line_abs_err_Txx"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Error (absolute):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QCheckBox" name="checkBox_Txx">
        <property name="text">
         <string>Txx</string>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QCheckBox" name="checkBox_Tyy">
        <property name="text">
         <string>Tyy</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QLineEdit" name="line_abs_err_Txz"/>
      </item>
      <item row="0" column="7">
       <widget class="QCheckBox" name="checkBox_Txz">
        <property name="text">
         <string>Txz</string>
        </property>
       </widget>
      </item>
      <item row="3" column="10">
       <widget class="QLineEdit" name="line_abs_err_Txy"/>
      </item>
      <item row="2" column="10">
       <widget class="QLineEdit" name="line_rel_err_Txy"/>
      </item>
      <item row="2" column="6">
       <widget class="QLineEdit" name="line_rel_err_Tzz"/>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_V">
        <property name="text">
         <string>V</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="checkBox_gy">
        <property name="text">
         <string>gy</string>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QLineEdit" name="line_rel_err_Txx"/>
      </item>
      <item row="2" column="11">
       <widget class="QLineEdit" name="line_rel_err_Tyy"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="line_rel_err_gz"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Order in the data file</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="line_order_V">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="line_order_gz"/>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="line_order_gx"/>
      </item>
      <item row="4" column="4">
       <widget class="QLineEdit" name="line_order_gy"/>
      </item>
      <item row="4" column="6">
       <widget class="QLineEdit" name="line_order_Tzz">
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QLineEdit" name="line_order_Txz"/>
      </item>
      <item row="4" column="8">
       <widget class="QLineEdit" name="line_order_Tyz"/>
      </item>
      <item row="4" column="9">
       <widget class="QLineEdit" name="line_order_Txx"/>
      </item>
      <item row="4" column="10">
       <widget class="QLineEdit" name="line_order_Txy"/>
      </item>
      <item row="4" column="11">
       <widget class="QLineEdit" name="line_order_Tyy"/>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="horizontalGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>393</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Observation Height (m):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="observation_height_lineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="generate_btn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>790</y>
      <width>301</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Configuration Files</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gridGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>1171</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Study region</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_7"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of padding cells on one side&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLineEdit" name="lineEdit_18"/>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLineEdit" name="lineEdit_15"/>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Increasing factor for the padding cell sizes</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>y:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEdit_8"/>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="lineEdit_9"/>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="lineEdit_12"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>num</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>z</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLineEdit" name="lineEdit_13"/>
     </item>
     <item row="5" column="5">
      <widget class="QLineEdit" name="lineEdit_14"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="lineEdit_10"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
